docs/how-to/profiling.txt

```diff
@@ -0,0 +1,95 @@
+.. _howto-profiling:
+
+#################################
+ How To: Profile An Experiment
```

Comment:
I think this page naming convention is useful, particularly for search, but it is inconsistent with the rest of the pages so maybe we should replace the title.

Reply:
you mean the prefix of "how to"?

Reply:
Yeah